includes/admin/class-wcmp-country-codes.php

```diff
@@ -8,29 +8,55 @@
     return new WCMP_Country_Codes();
 }
 
-use MyParcelNL\Sdk\src\Model\Consignment\AbstractConsignment;
-
 class WCMP_Country_Codes
 {
+    private const EURO_COUNTRIES = [
```

Comment:
Kunnen we dit niet uit de SDK halen? Of juist in de SDK stoppen? Gebruiken we dit ook in Magento?

Reply:
Wij hebben in de SDK ook de method isEuCountry. Deze is alleen nog niet zo gemaakt dat het voor nu werkbaar is voor de WooCommerce plugin.

Ik heb een story gemaakt om het in de SDK aan te passen zodat het in alle situaties bruikbaar is. Zodra het is aangepast dan zal het ook in de WooCommerce plugin worden verwerkt.